### PR TITLE
zh-trans:blog/_posts/2019-11-05-kubernetes-with-microk8s.md

### PR DESCRIPTION
**Pull request by Aresforchina**
zh-trans:blog/_posts/2019-11-05-kubernetes-with-microk8s.md
ref:https://github.com/k8smeetup/website-tasks/issues/3670

### CODE REVIEW

File: content/zh/blog/_posts/2019-11-05-kubernetes-with-microk8s.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
+---
+title：’使用 Microk8s 在 Linux 上本地运行 Kubernetes‘
```
Comment (tanjunchen) [inline content/zh/blog/_posts/2019-11-05-kubernetes-with-microk8s.md]: image

title 中的符号有问题

File: content/zh/blog/_posts/2019-11-05-kubernetes-with-microk8s.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+-->
+**Authors**: [Ihor Dvoretskyi](https://twitter.com/idvoretskyi), Developer Advocate, Cloud Native Computing Foundation; [Carmine Rimi](https://twitter.com/carminerimi)
+<!--
+This article, the second in a [series](/blog/2019/03/28/running-kubernetes-locally-on-linux-with-minikube-now-with-kubernetes-1.14-support/) about local deployment options on Linux, and covers [MicroK8s](https://microk8s.io/). Microk8s is the click-and-run solution for deploying a Kubernetes cluster locally, originally developed by Canonical, the publisher of Ubuntu.
+-->
+本文是关于 Linux 上的本地部署选项的 [系列](https://twitter.com/idvoretskyi) 文章的第二篇，其中介绍了 [MicroK8s](https://microk8s.io/)。Microk8s 是本地部署 Kubernetes 集群的点击运行解决方案，最初由 Ubuntu 的发布者 Canonical 开发。
+<!--
+While Minikube usually spins up a local virtual machine (VM) for the Kubernetes cluster, MicroK8s doesn’t require a VM. It uses [snap](https://snapcraft.io/) packages, an application packaging and isolation technology.
+-->
+虽然 Minikube 通常为 Kubernetes 集群生成一个本地虚拟机（VM），但是 MicroK8s 不需要 VM。它使用[snap](https://snapcraft.io/) 包，这是一种应用程序打包和隔离技术。
```
Comment (xichengliudui) [inline content/zh/blog/_posts/2019-11-05-kubernetes-with-microk8s.md]:
```suggestion
虽然 Minikube 通常为 Kubernetes 集群创建一个本地虚拟机（VM），但是 MicroK8s 不需要 VM。它使用 [snap](https://snapcraft.io/) 包，这是一种应用程序打包和隔离技术。
```

File: content/zh/blog/_posts/2019-11-05-kubernetes-with-microk8s.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+<!--
+This difference has its pros and cons. Here we’ll discuss a few of the interesting differences, and comparing the benefits of a VM based approach with the benefits of a non-VM approach. One of the first factors is cross-platform portability. While a Minikube VM is portable across operating systems - it supports not only Linux, but Windows, macOS, and even FreeBSD - Microk8s requires Linux, and only on those distributions [that support snaps](https://snapcraft.io/docs/installing-snapd). Most popular Linux distributions are supported. 
+-->
+这种差异有其优点和缺点。在这里，我们将讨论一些有趣的区别，并且基于 VM 的方法和非 VM 方法的好处。第一个因素是跨平台的移植性。虽然 Minikube VM 可以跨操作系统移植——它不仅支持 Linux，还支持 Windows、macOS、甚至 FreeBSD，但 Microk8s 需要 Linux，而且只在 [那些支持 snaps](https://snapcraft.io/docs/installing-snapd) 的发行版上。支持大多数流行的 Linux 发行版。
```
Comment (xichengliudui) [inline content/zh/blog/_posts/2019-11-05-kubernetes-with-microk8s.md]:
```suggestion
这种差异有其优点和缺点。在这里，我们将讨论一些有趣的区别，并且基于 VM 的方法和非 VM 方法的好处。第一个因素是跨平台的移植性。虽然 Minikube VM 可以跨操作系统移植——它不仅支持 Linux，还支持 Windows、macOS、甚至 FreeBSD，但 Microk8s 需要 Linux，而且只在[那些支持 snaps](https://snapcraft.io/docs/installing-snapd) 的发行版上。支持大多数流行的 Linux 发行版。
```

File: content/zh/blog/_posts/2019-11-05-kubernetes-with-microk8s.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+-->
+另一个考虑到的因素是资源消耗。虽然 VM 设备为您提供了更好的可移植性，但它确实意味着您将消耗更多资源来运行 VM，这主要是因为 VM 提供了一个完整的操作系统，并且运行在管理程序之上。当 VM 处于休眠时你将消耗更多的磁盘空间。当它运行时，你将会消耗更多的 RAM 和 CPU。因为 MIcrok8s 不需要创建虚拟机，你将会有更多的资源去运行你的工作负载和其他设备。考虑到所占用的空间更小，MIcroK8s 是物联网设备的理想选择-你甚至可以在 Paspberry Pi 和设备上使用它！
+<!--
+Finally, the projects appear to follow a different release cadence and strategy. MicroK8s, and snaps in general provide [channels](https://snapcraft.io/docs/channels) that allow you to consume beta and release candidate versions of new releases of Kubernetes, as well as the previous stable release. Microk8s generally releases the stable release of upstream Kubernetes almost immediately.
+-->
+最后，项目似乎遵循了不同的发布节奏和策略。Microk8s 和 snaps 通常提供 [渠道](https://snapcraft.io/docs/channels) 允许你使用测试版和发布 KUbernetes 新版本的候选版本，同样也提供先前稳定版本。Microk8s 通常几乎立刻发布 Kubernetes 上游的稳定版本。
```
Comment (xichengliudui) [inline content/zh/blog/_posts/2019-11-05-kubernetes-with-microk8s.md]:
```suggestion
最后，项目似乎遵循了不同的发布节奏和策略。Microk8s 和 snaps 通常提供[渠道](https://snapcraft.io/docs/channels)允许你使用测试版和发布 KUbernetes 新版本的候选版本，同样也提供先前稳定版本。Microk8s 通常几乎立刻发布 Kubernetes 上游的稳定版本。
```

File: content/zh/blog/_posts/2019-11-05-kubernetes-with-microk8s.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+-->
+但是等等，还有更多！Minikube 和 Microk8s 都是作为单节点集群启动的。本质上来说，它们允许你用单个工作节点创建 Kubernetes 集群。这种情况即将改变 - MicroK8s 早期的 alpha 版本包括集群。有了这个能力，你可以创建正如你希望多的工作节点的 KUbernetes 集群。对于创建集群来说，这是一个没有主见的选项 - 开发者在节点之间创建网络连接和集成了其他所需要的基础设施，比如一个外部的负载均衡。总的来说，MicroK8s 提供了一种快速简易的方法，使得少量的计算机和虚拟机变成一个多节点的 Kubernetes 集群。以后我们将撰写更多这种体系结构的文章。
+<!--
+## Disclaimer
+
+This is not an official guide to MicroK8s. You may find detailed information on running and using MicroK8s on it's official [webpage](https://microk8s.io/docs/), where different use cases, operating systems, environments, etc. are covered. Instead, the purpose of this post is to provide clear and easy guidelines for running MicroK8s on Linux.
+-->
+## 免责声明 
+
+这不是 MicroK8s 官方介绍文档。你可以在它的官方 [网页](https://microk8s.io/docs/) 查询运行和使用 MicroK8s 的详情信息，其中覆盖了不同的用例，操作系统，环境等。相反，这篇文章的意图是提供在 Linux 上运行 MicroK8s 清晰易懂的指南。
```
Comment (xichengliudui) [inline content/zh/blog/_posts/2019-11-05-kubernetes-with-microk8s.md]:
```suggestion
这不是 MicroK8s 官方介绍文档。你可以在它的官方[网页](https://microk8s.io/docs/)查询运行和使用 MicroK8s 的详情信息，其中覆盖了不同的用例，操作系统，环境等。相反，这篇文章的意图是提供在 Linux 上运行 MicroK8s 清晰易懂的指南。
```

File: content/zh/blog/_posts/2019-11-05-kubernetes-with-microk8s.md [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+<!--
+## Prerequisites
+
+A Linux distribution that [supports snaps](https://snapcraft.io/docs/installing-snapd), is required. In this guide, we’ll use Ubuntu 18.04 LTS, it supports snaps out-of-the-box.
+If you are interested in running Microk8s on Windows or Mac, you should check out [Multipass](https://multipass.run) to stand up a quick Ubuntu VM as the official way to run virtual Ubuntu on your system.
+-->
+## 前提条件 
+
+一个 [支持 snaps](https://snapcraft.io/docs/installing-snapd) 的 Linux 发行版是被需要的。这篇指南，我们将会用支持 snaps 且即开即用的 Ubuntu 18.04 LTS。如果你对运行在 Windows 或者 Mac 上的 MicroK8s 感兴趣，你应该检查 [多通道](https://multipass.run)，安装一个快速的 Ubuntu VM，作为在你的系统上运行虚拟机 Ubuntu 的官方方式。
```
Comment (xichengliudui) [inline content/zh/blog/_posts/2019-11-05-kubernetes-with-microk8s.md]:
```suggestion
一个[支持 snaps](https://snapcraft.io/docs/installing-snapd) 的 Linux 发行版是被需要的。这篇指南，我们将会用支持 snaps 且即开即用的 Ubuntu 18.04 LTS。如果你对运行在 Windows 或者 Mac 上的 MicroK8s 感兴趣，你应该检查[多通道](https://multipass.run)，安装一个快速的 Ubuntu VM，作为在你的系统上运行虚拟机 Ubuntu 的官方方式。
```

Comment (Aresforchina) [reply content/zh/blog/_posts/2019-11-05-kubernetes-with-microk8s.md]: 我已经修改它们，感谢细心review。

**Pull request merged**